aea/cli/create.py

```diff
@@ -129,44 +132,55 @@ def _create_aea(
         raise click.ClickException(str(e))
 
 
-@click.command()
-@click.argument("agent_name", type=str, required=True)
-@click.option(
-    "--author",
-    type=str,
-    required=False,
-    help="Add the author to run `init` before `create` execution.",
-)
-@click.option("--local", is_flag=True, help="For using local folder.")
-@click.option("--empty", is_flag=True, help="Not adding default dependencies.")
-@click.pass_context
-def create(click_context, agent_name, author, local, empty):
-    """Create an agent."""
-    try:
-        _check_is_parent_folders_are_aea_projects_recursively()
-    except Exception:
-        raise click.ClickException(
-            "The current folder is already an AEA project. Please move to the parent folder."
-        )
+def _crete_agent_config(ctx: Context, agent_name: str, set_author: str) -> AgentConfig:
```

Comment:
typo

Reply:
Feels like this could be a helper method in config and imported here;

Reply:
I'm not sure it will be used anywhere else.